content/kubermatic/master/_index.en.md

```diff
@@ -9,30 +9,73 @@ date = 2019-04-27T16:06:34+02:00
+KKP provides two different levels of Monitoring, Logging, and Alerting. 
 
-### Update Kubernetes
+1. The first targets only the management components (master, seed, CRDs) and is independent. This is the Master/Seed Cluster MLA Stack and only the KKP Admins can access this monitoring data.
 
-KKP provides live updates of your Kubernetes cluster without disrupting your daily business. Use all new features of Kubernetes as you increase scale.
+2. The other component is the User Cluster MLA Stack which is a true multi-tenancy solution for all your end-users as well as a comprehensive overview for the KKP Admin. It helps to speed up individual progress but lets the Admin keep an overview of the big picture. It can be configured per seed to match the requirements of the organizational structure All users can access monitoring data of the user clusters under the projects that they are members of.
 
-### Scale Your Cluster
+Integrated Monitoring, Logging and Alerting functionality for applications and services in KKP user clusters are built using Prometheus, Loki, Cortex and Grafana. Furthermore, this can be enabled with a single click on the KKP UI.
+
+#### OIDC Provider Configuration
+Since Kubernetes does not provide an OpenID Connect (OIDC) Identity Provider, KKP allows the user to configure a custom OIDC. This way you can grant access and information to the right stakeholders.
+
+#### Upgrading Control Plane and the kubelets
+A specific version of Kubernetes’ control plane typically supports a specific range of kubelet versions connected to it. KKP enforces the rule “kubelet must not be newer than kube-apiserver, and maybe up to two minor versions older” on its own. KKP ensures this rule is followed by checking during each upgrade of the clusters’ control plane or node’s kubelet. Additionally, only compatible versions are listed in the UI as available for upgrades.
+
+#### OPA
+To enforce policies and improve governance in Kubernetes, the Open Policy Agent (OPA) can be used. KKP integrates it using Gatekeeper which is an OPA’s Kubernetes native policy engine. As an Admin you can enable and enforce OPA integration during cluster creation by default via the UI.
+
+#### Cluster Templates
+Clusters can be created in a few clicks with the UI. To take the user experience one step further and make repetitive tasks redundant, cluster templates allow you to save data entered into a wizard to create multiple clusters from a single template at once. The templates can be saved to be used subsequently for new cluster creation. 
+
+#### Use default Addons to extend the functionality of Kubernetes
+Addons are specific services and tools extending the functionality of Kubernetes. Default addons are installed in each user cluster in KKP. The KKP Operator comes with a tool to output full default KKP configuration, serving as a starting point for adjustments. Accessible addons can be installed in each user cluster in KKP on user demand.
```

Comment:
I would probably drop this because we already have a lot of points and this is IMO not so worthwhile highlighting.